packages/test/snapshots/README.md

```diff
@@ -35,36 +35,43 @@ npm run test
 
 For submitting changes to the snapshot test content, follow these steps:
 
--   Submitting the changes to the test content:
+-   Submit changes to the test content:
+
     -   Sync the content for the snapshot tests as mentioned [here](#Syncing-snapshot-tests-content).
-    -   Go to the `content` subdirectory, create a new branch and update the snapshot content. Ideally, you should not update the content manually but follow one of these:
+    -   Go to the `content` subdirectory, create a new branch and update the snapshot content. DO NOT update the content manually. You should follow one of these:
         -   [Adding new snapshots](#Adding-new-snapshots-to-the-repo) - If you are adding new snapshots.
         -   [Updating existing snapshots](#Updating-existing-snapshots-in-the-repo) - If you updating existing snapshots.
     -   Create a commit and push the new branch.
     -   Create a PR against `main` branch in the [FluidFrameworkTestData](https://github.com/microsoft/FluidFrameworkTestData) repo. Ensure the build passes, get the changes approved and merge the PR.
     -   Checkout `main` branch of FluidFrameworkTestData repo, i.e., in the `content/` folder and run `git pull` to ensure it has your new changes.
--   Updating FluidFramework repo to point to the new content:
 
-    -   Go to the FluidFramework repo root. Now you can run `git submodule update --init --recursive` in repo root to sync your changes with the newly updated test repo.
-    -   `git status` should show changes under `packages/test/snapshots/content`. For example:
+    > Note: The snapshots in the FluidFrameworkTestData repo are exactly what is generated by the new / update scripts. They should not be modified manually. The idea is to preserve these snapshots as-is. The tests may normalize these snapshots when comparing / validating them but the original snapshot is not normalized or modified.
```

Comment:
Could be nice to have a reference to `snapshotNormalizer.ts` somewhere in this README

Reply:
Good point. Added a section for it. Please take a look.